docs/Getting Started/cli.md

```diff
@@ -66,12 +56,8 @@ jobs:
 The repository contains examples which you can use to test Codyze. 
 Below are the commands to call Codyze on these examples.
 
-```shell title="Codyze v2"
-./gradlew :codyze-v2:run --args="-c -s ../examples/botan/blockciphers/Prudkovskiy.Qt_LockBox/crypto.cpp -m ../examples/botan/MARK"
-```
-
-```shell title="Codyze v3"
-./gradlew :codyze-v3:codyze:run --args="analyze -s ../../examples/botan/blockciphers/Prudkovskiy.Qt_LockBox/crypto.cpp --spec ../../examples/botan/MARK"
+```shell
+./gradlew :codyze-cli:run --args="analyze -s ../examples/botan/blockciphers/Prudkovskiy.Qt_LockBox/crypto.cpp --spec ../examples/botan/MARK"
```

Comment:
I would remove this example as we do not have MARK support in v3.

We could instead refer to our test resources, maybe.

Reply:
Done in f1738c413119693e02d2122ea98faa8bc277fc27.